ArduPlane/tiltrotor.cpp

```diff
@@ -198,14 +198,23 @@ void QuadPlane::tiltrotor_update(void)
 
   By applying _tilt_equal_thrust the tilted motors effectively become
   a single pitch control motor.
+
+  Note that we use a different strategy for when we are transitioning
+  to VTOL as compared to from VTOL flight. The reason for that is we
```

Comment:
Nitpicking: the sentence might look better as _Note that we use a different strategy for when we are transitioning **into** VTOL as compared to from VTOL flight_. At least it was confusing to me reading the way it is now.